### PR TITLE
Bugfix FXIOS-6006 [v115] wrong text displayed in Touch ID Biometrics

### PR DESCRIPTION
**Pull request by petruSt24**
#13636

[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6006)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/13636)

### Description
Added the text for Fingerprint and FaceID biometric authentication errors

### Pull requests checks where applicable
- [X] Fill in the three TODOs above (tickets number and description of your work)
- [X] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [ ] Unit tests written and passing
- [ ] Documentation / comments for complex code and public methods

### CODE REVIEW

**Review by yoanarios — COMMENTED** [review]
LGTM but I will wait for @nishant2718 approval because I think he has more context about this ticket

File: Client/Frontend/AuthenticationManager/AppAuthenticator.swift
Comment (nishant2718) [inline Client/Frontend/AuthenticationManager/AppAuthenticator.swift:47]: We've tried this approach before and it's led to the ticket now. The reason we can't do it this way is because when biometric authentication fails, the description can't be updated (see the last screenshot in the github issue). 

What we actually need is a more generalized string that can suffice as a description for both when a user is attempting biometric auth, and when that fails and asked to put a passcode.

Comment (petruSt24) [reply Client/Frontend/AuthenticationManager/AppAuthenticator.swift:47]: Interesting insight, so should I put something like "Secure access requires biometric authentication. If unsuccessful, please provide your passcode when prompted." to cover all the cases or ask someone from Design/UX to help with the string?

**Comment by dnarcese** [comment]
This needs a new string. It needs to be taken out of 114

File: Client/Frontend/Strings.swift
Comment (lmarceau) [inline Client/Frontend/Strings.swift:89]: We need to update the string key to be able to export the new string for localisation.
```suggestion
                key: "Biometry.Screen.UniversalAuthenticationReason.v115",
```

**Review by lmarceau — APPROVED** [review]

**Review by nishant2718 — APPROVED** [review]

**Review by yoanarios — APPROVED** [review]

**Pull request merged**
